tests/eval_files/trio910.py

```diff
@@ -510,6 +511,16 @@ async def foo_test2():
     print("...")
 
 
+@custom_disabled_decorator
+async def foo_ARG_test():
+    print("...")
+
+
+@custom_disabled_decorator()
+async def foo_ARG_test2():
+    print("...")
+
+
```

Comment:
idr if this actually covered more, but no reason not to have a test for it in the eval file